package.json

```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "babel-cli": "^6.11.4",
     "classnames": "^2.2.3",
+    "cross-env": "^2.0.0",
```

Comment:
I know it's weird, but I was hoping to restrict `dependencies` to the core reactstrap library dependencies. I must have accidently added `babel-cli` to it.